internal/common/database/migrations_test.go

```diff
@@ -0,0 +1,46 @@
+package database
+
+import (
+	"testing"
+	"strconv"
+	"strings"
+	"io/fs"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestReadMigrations(t *testing.T) {
+	mockFiles := []string{
```

Comment:
This is a good start - using a mocked filesystem with `io/fs` is good. Try adding some more files in here, with varying differences to match (or not) the expected patterns) - see the existing migrations in `internal/scheduler/database/migrations/*.sql` and also in `internal/lookoutv2/schema/migrations/*.sql`.

Reply:
Sure, will look into it